packages/grid/x-data-grid/src/components/cell/GridCell.tsx

```diff
@@ -681,13 +322,21 @@ const GridCellV7 = React.forwardRef<HTMLDivElement, GridCellV7Props>((props, ref
       };
     }
     const cellStyle = {
-      minWidth: width,
```

Comment:
I think I just discovered the reason for `minWidth` here - it's for column resizing:


https://github.com/mui/mui-x/assets/13808724/00a03754-d33d-4898-a473-d9443dfb35ee

Reply:
I've kept the CSS var but restored the min/max width.